src/deadline/job_attachments/api/manifest.py

```diff
@@ -247,3 +260,182 @@ def process_output(status: FileStatus, path: str, output_diff: ManifestDiff):
             process_output(fast_diff_item[1], fast_diff_item[0], output)
 
     return output
+
+
+def _manifest_upload(
+    manifest_file: str,
+    s3_bucket_name: str,
+    manifest_path: str,
+    s3_metadata: Dict[str, Any],
+    boto_session: boto3.Session,
+    logger: ClickLogger = ClickLogger(False),
+):
+    """
+    BETA API - This API is still evolving but will be made public in the near future.
+    API to upload a job attachment manifest to the Content Address Storage. Manifests will be
```

Comment:
nit: should this be `Content Addressable Storage` to be consistent with CLI help text/docstring?
```suggestion
    API to upload a job attachment manifest to the Content Addressable Storage. Manifests will be
```

Reply:
Agreed!